src/Types.jl

```diff
@@ -1351,13 +1422,37 @@ function project_key_order(key::String)
     return 8
 end
 
+string(x::Vector{String}) = x
```

Comment:
Ok, I should have caught this the first time around but this is some pretty bad type piracy.

Cross ref: https://github.com/JuliaLang/julia/issues/30945#issuecomment-460065670

Still exists in Pkg, needs to be removed.